widgets/src/main/resources/org/pentaho/gwt/widgets/themes/public/themes/ruby/globalRuby.css

```diff
@@ -5063,6 +5063,11 @@ div.dijitDateTextBox.dijitComboBox {
   width: 100%;
 }
 
+.fileChooserCellFileName:focus-visible {
+  outline: var(--focus-ring);
+  outline-offset: -2px;
+}
```

Comment:
Apparently, all that is needed here is: `--focus-ring-offset: -2px;`.

Reply:
thanks for the suggestion. Addressing it.